src/Kunstmaan/GeneratorBundle/Resources/SensioGeneratorBundle/skeleton/config/security.yaml

```diff
@@ -0,0 +1,41 @@
+acl:
+    connection: default
+
+security:
+    encoders:
+        FOS\UserBundle\Model\UserInterface: sha512
```

Comment:
I would suggest changing the default algoritm to `bcrypt` as it the best we can use in php 7.1.

Reply:
Yes that's a good improvement